src/lib/containers/UserSearchBoxV2.tsx

```diff
@@ -24,6 +25,7 @@ export type UserSearchBoxProps = {
   typeFilter?: TYPE_FILTER
   filterPredicate?: (item: UserGroupHeader) => boolean
   placeholder?: string
+  focusOnSelect?: boolean
```

Comment:
In case there are multiple SearchBoxes and do not want to focus on any or focus on one.